docs/source/core/embedding.rst

```diff
@@ -226,4 +226,55 @@ workload identity, etc…)
 
     [print(e.embeddings, e) for e in elements]
     print(query_embedding, query)
-    print(embedding_encoder.is_unit_vector(), embedding_encoder.num_of_dimensions())
\ No newline at end of file
+    print(embedding_encoder.is_unit_vector(), embedding_encoder.num_of_dimensions())
+
+``VoyageAIEmbeddingEncoder``
```

Comment:
Our documentation has moved to https://docs.unstructured.io. We can do a separate PR into https://github.com/Unstructured-IO/docs/ with these updates. cc @cmscmadd @MKhalusova